### ISSUE TITLE
It doesn't sync XPG SSD m.2

### ISSUE THREAD

**Issue by mariocsantos** [ChromaControl/ChromaControl#53]
The motherboard synced, but the SSD no :(

![image](https://user-images.githubusercontent.com/13947440/112398222-6e8a7f80-8ce2-11eb-8313-cc242328a53d.png)

**Comment by AndrewBabbitt97** [comment]
Please use Armory Crate and update to the latest device HALs. I don't do any active testing against the old Aura installer.

**Comment by mariocsantos** [comment]
I tried to use Armory Crate, and it doesn't work, with old version works, strange. 🤔

I just installed XPG software and it works perfectally! 🙌

Thanks for the quick response.

**Comment by suvamrox** [comment]
Bro I install XPG software, still not working. plz help me

**Comment by mariocsantos** [comment]
The SSD sync is incosistent, sometimes it fails, sometimes works. So, I decided to reinstall windows.

I tried the bellow steps, and it works perfectally again.

- Install Razer Synapse
- Install Armory Crate
- Install the old version of Chroma control, [AuraConnect](https://www.mediafire.com/file/pqbz0k1ml0aaigv/AuraConnectSetup.exe/file).

I recommend you to reinstall windows too.

Sorry for answering so late.
